release-tools/build.make

```diff
@@ -57,12 +57,30 @@ else
 TESTARGS =
 endif
 
+BUILD_IN_DOCKER?=false
+DOCKER_BUILDER_WD=/go/src/github.com/kubernetes-csi/external-attacher
+DOCKER_BUILDER_IMG=golang
```

Comment:
travis.yaml contains the authoritative definition of which Go version is meant to be used for building the source. It would be useful to use that information to pick the right image - not sure whether that is possible, though.

Reply:
Lemme take a look; I guess this is going to need some `$(shell...` magic though. Unsure how reliable `grep` could be here...